test/common/fixtures.js

```diff
@@ -15,8 +15,13 @@ function readFixtureSync(args, enc) {
   return fs.readFileSync(fixturesPath(args), enc);
 }
 
-function readFixtureKey(name, enc) {
-  return fs.readFileSync(fixturesPath('keys', name), enc);
+function readFixtureKey(name, enc, trim) {
+  let result = fs.readFileSync(fixturesPath('keys', name), enc);
+  if (trim) {
```

Comment:
If taking the previous suggestion:

```suggestion
  if (options.trim) {
```